docs/source/features/quantization/quark.md

```diff
@@ -0,0 +1,218 @@
+(quark)=
+
+# QUARK
+
+Quantization can effectively reduce memory and bandwidth usage, accelerate computation and improve
+throughput while with minimal accuracy loss. vLLM can leverage [Quark](https://quark.docs.amd.com/latest/),
+the flexible and powerful quantization toolkit, to produce performant quantized models to run on GPUs
+of AMD. Quark has specialized support for quantizing large language models with weight,
+activation and kv-cache quantization and along with the cutting-edge quantization algorithms like
+AWQ, GPTQ, Rotation and SmoothQuant.
+
+## Quark Installation
+
+Before quantizing model, you need to install Quark. Release of Quark can be installed with pip:
```

Comment:
"Before quantizing a model" or "quantizing models".

"Release of Quark" -> "The latest release of Quark"